ci/Jenkinsfiles/export.groovy

```diff
@@ -183,14 +183,14 @@ pipeline {
             script {
               def moduleDir = 'docker/nuxeo-explorer-export-docker'
               def oneLineClid = getClid("${INSTANCE_CLID}")
-              sh """#!/bin/bash +x
-                CONNECT_EXPLORER_CLID="${oneLineClid}" \
-                CONNECT_EXPORT_CLID="${oneLineClid}" \
+              sh '''#!/bin/bash +x
+                CONNECT_EXPLORER_CLID=$oneLineClid \
+                CONNECT_EXPORT_CLID=$oneLineClid \
```

Comment:
Not sure about these 2 as `oneLineClid` is a Groovy variable, it probably needs to be interpolated.
Keeping the existing code should work and not trigger the security warning.

Reply:
I do get a warning on [INSTANCE_CLID], so maybe it's about changing
```
def oneLineClid = getClid("${INSTANCE_CLID}")
```
into 
```
def oneLineClid = getClid($INSTANCE_CLID)
```

?

Reply:
Alright thanks :+1: 

Aren't the double quotes needed for proper resolution of `${moduleDir}` ?

Reply:
yes!